circle/service/user_service.go

```
@@ -0,0 +1,187 @@
+	m:=config.Email
+	html := "<h1>验证码：" + VerificationCode + "</h1>"
+	e := email.NewEmail()
+	e.From = "luohuixi <2388287244@qq.com>"    
+	e.To = []string{ee}         
+	e.Subject = "验证码"                            
+	e.Text = []byte("This is a plain text body.") 
+	e.HTML = []byte(html)                    
+	smtpHost := "smtp.qq.com"                                           
+	smtpPort := "587"                                                             
+	auth := smtp.PlainAuth("", "2388287244@qq.com", m, smtpHost)
+	e.Send(smtpHost+":"+smtpPort, auth)  
```

golangci-lint (1.62.2)

Error return value of `e.Send` is not checked

(errcheck)